tests/FSharp.Core.UnitTests/FSharp.Core.UnitTests.fsproj

```diff
@@ -38,6 +38,7 @@
     <Compile Include="FSharp.Core\ComparersRegression.fs" />
     <Compile Include="FSharp.Core\DiscriminatedUnionType.fs" />
     <Compile Include="FSharp.Core\RecordTypes.fs" />
+    <Compile Include="FSharp.Core\RefStructs.fs" Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'" />
```

Comment:
It's rather a nitpick, but I would rather put the test as a whole into `#if NETCOREAPP`, than specifying it in the project file, it's bit more visible when you're looking at it. Other than that - looks good to me.

Reply:
Fixed.